### ISSUE TITLE
Doc precision

### ISSUE THREAD

**Issue by esteem8** [steemit/steem-js#93]
Provide a more precise methods documentation.

#### Expected behavior

* Method name

* Method description

* Method code with parameters

* Method parameters detail

#### Actual behavior

Method name

Method code with parameters

#### How

B.E.

### Get Followers

Return account followers.

```
steem.api.getFollowers(following, startFollower, followType, limit, function(err, result) {
  console.log(err, result);
});
```
```
following: [String] -> 'fabien'
startFollower: [String] -> '' || 'fabien'
followType: [?] -> null
limit: [Number] -> '0', '100'
```

**Comment by esteem8** [comment]
I think it would be easier to take in hand the API.

An example could also be great.

If this is accepted, I will do it when I will have the time.

**Comment by esteem8** [comment]
A link to codepen.io could also be great.

Eventually using [there api]( https://blog.codepen.io/documentation/api/prefill/).

**Comment by adcpm** [comment]
I agree, that would be great to have more information on the documentation. If you do it we will accept your PR. @good-karma been working on something similar Steem API tester:  https://esteemapp.github.io/steemapi/#!/Account/get_get_followers. There is some descriptions and default parameters of the methods, this can help for building Steem.js doc.

**Comment by marvin-we** [comment]
Hello together,

I just saw this ticket and I guess this is a problem that everyone providing an API wrapper is fighting with. I develop [SteemJ](https://github.com/marvin-we/steem-java-api-wrapper) which is the Steem API wrapper for Java and I also had to start documenting everything in the Wiki. I guess this is the same case of steem-python and radiator (the api wrapper for ruby) and all the other wrapper too. 

For sure fields are accessed in a different way in each programming language, but the available fields are the same for all of us as we all work against the Steem api. Therefore, what do you think of having a centralized documentation for the api? I guess this would help us, the developers, and the users for sure...

**Comment by muksihs** [comment]
As steem is moving towards https:// instead of wss:// why not use swagger/openapi so that multiple client language bindings can be generated automatically?

**Comment by Stormrose** [comment]
I'm going to put some time into improving the documentation. In the short term by hacking at the doc/README/md file. I don't expect to complete the API documentation, but I can at least make a start.

@timcliff on steemit has expressed interest in updating central developer docs. The best place for swagger/openapi specs is where the API is published, not within a language binding project like steem-js.

**Comment by ety001** [comment]
This repository has been superseded by a complete 2025 rewrite. The new trunk (`master`, starting at v1.1.0) is a ground-up rewrite of `@steemit/steem-js` and the legacy 0.7.x / 0.8.x code line is archived.

As part of that transition, this issue is being closed because:
- It was filed against the legacy codebase, and the code it references no longer exists in the rewritten trunk.
- The new codebase reimplements the affected areas (RPC routing, transaction serialization, auth/signing, crypto, browser builds) from scratch, so this issue cannot be carried over as-is.

If the problem or request still applies to the rewritten codebase (v1.1.0+ on `master`), please open a new issue against the current trunk with details that reproduce on the new version. Thank you for taking the time to report this originally.
